app/constants/types.js

```diff
@@ -2,3 +2,5 @@ export const SET_CURRENT_SERVER = 'SET_CURRENT_SERVER';
 export const SET_ALL_SETTINGS = 'SET_ALL_SETTINGS';
 export const SET_CUSTOM_EMOJIS = 'SET_CUSTOM_EMOJIS';
 export const ADD_SETTINGS = 'ADD_SETTINGS';
+export const SHARE_MESSAGE = 'SHARE_MESSAGE';
```

Comment:
We need to remove this `app/constants/types` file in future.
I just never had a chance to do it (I'm not asking you to do it in this PR).
It's breaking the pattern.
Anyway, if you still need to create these consts, do it elsewhere.